indexing-service/src/test/java/io/druid/indexing/overlord/RemoteTaskRunnerTest.java

```diff
@@ -417,7 +418,9 @@ public boolean isValid()
               {
                 return remoteTaskRunner.getRemovedWorkerCleanups().isEmpty();
               }
-            }
+            },
+            // cleanup task is independently scheduled by event listener. we need to wait some more time.
+            config.getTaskCleanupTimeout().toStandardDuration().getMillis() * 2
```

Comment:
very strange that it would take more than 1 second. We can try with 2 seconds for travis